### DISCUSSION TITLE
probleme raspberry with firebase

### DISCUSSION THREAD

**Discussion by khawla-nasfi** [supabase/supabase#6844] [Questions]
hello
can I ask a question about my IoT project? to be honest, I am having a problem synchronizing my firebase with my connected object (raspberry pi 3 is connected in serial with Arduino Uno ).would you please help me?

**Comment by jaitaiwan** [comment]
ARM isn’t a supported platform for supabase deployment [yet. ](https://github.com/supabase/supabase/issues/2954)
